src/js/renderables/allies/adult.js

```diff
@@ -6,25 +6,14 @@ class AdultEntity extends AllyEntity {
 
     constructor(x, y, settings) {
```

Comment:
If you are updating line 9 to current change, remove settings from constructor and update anything that is called new Adult().